build.zig.zon

```diff
@@ -3,10 +3,14 @@
 	.version = "0.0.0",
 	.paths = .{""},
 	.dependencies = .{
+		.deps_local = .{
+			.url = "file:../Cubyz-libs/zig-out.tar.gz", // macos has no official deps release
+			.hash = "12209f7db8911ff57c69db2097437ad5e00141e862b0e760926c966b8652d09edf43",
+		},
 		.deps = .{
-			//.url = "file:///home/mint/Cubyz-libs/zig-out.tar.gz", // fetching the lib from the local folder
+			// url = "file:../Cubyz-libs/zig-out.tar.gz", // fetching the lib from the local folder
```

Comment:
Could you please keep the dot in front of the `url` field? Makes it easier to swap the two cases.

Reply:
Updated. But does it work?

Reply:
Yeah, it appears to work.